app/json/slate_configs.json

```diff
@@ -89,6 +89,24 @@
       }
     ]
   },
+  {
+    "id": "f99178fb-6bd0-4fa1-8109-cda181b697f6",
```

Comment:
I literally went to a uuid generator and got this. These aren't indexed in a db or anything are they? I'm 99% sure know but I'd like confirmation from The Data Guy

Reply:
the answer is yes